tests/suites/test_suite_psa_crypto.function

```diff
@@ -3700,8 +3730,14 @@ void signature_size( int type_arg,
 {
     psa_key_type_t type = type_arg;
     psa_algorithm_t alg = alg_arg;
-    size_t actual_size = PSA_ASYMMETRIC_SIGN_OUTPUT_SIZE( type, bits, alg );
+    size_t actual_size = PSA_SIGN_OUTPUT_SIZE( type, bits, alg );
+
     TEST_EQUAL( actual_size, (size_t) expected_size_arg );
+#if defined(MBEDTLS_TEST_DEPRECATED)
+    TEST_EQUAL( actual_size,
+                PSA_ASYMMETRIC_SIGN_OUTPUT_SIZE( type, bits, alg ) );
```

Comment:
This is conditioned by `MBEDTLS_TEST_DEPRECATED`. Using a deprecated macro outside of a `MBEDTLS_TEST_DEPRECATED` guard should result in a CI failure in the job with `MBEDTLS_DEPRECATED_WARNING` set.